/kiss_icp_matcher_node:
  ros__parameters:
    base_frame_id: "base_link"

    displacement: 1.0

    max_range: 100.0

    min_range: 1.0

    deskew: false

    voxel_size: 1.0

    max_points_per_voxel: 20

    initial_threshold: 2.0

    min_motion_th: 0.1
